fab/inventory/enikshay

```diff
@@ -12,13 +12,13 @@
 [formplayer0] # formplayer, riak
 172.25.2.7
 
-[formplayer1] # formplayer
+[formplayer1] # formplayer, riak
```

Comment:
interesting that riak is going to be sharing resources. do you know how much RAM will be on that machine?